lms/templates/instructor/edx_ace/allowedenroll/email/body.html

```diff
@@ -3,11 +3,11 @@
 {% load i18n %}
 {% load static %}
 {% block content %}
-<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation" style="box-shadow: 0px 0px 10px 2px #c8c8c8;">
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation" style="box-shadow: 0px 0px 10px 2px #c8c8c8; width: 100%;">
     <tr>
         <td style="width: 100%;
             max-width: 800px;
-            background-image: url(https://pearson-production-media.s3.amazonaws.com/IPbackground.svg);
+            background-image: url(https://partner-courses.pearson.com/static/pearson-theme/images/IPbackground.svg);
```

Comment:
This would fail if the openedx-themes PR is not pushed to prod. Could you please try by using Django's static tag? https://docs.djangoproject.com/en/5.2/howto/static-files/#configuring-static-files